pkg/microservice/aslan/config/consts.go

```diff
@@ -230,3 +230,10 @@ const (
 	CommonEnvCfgTypeSecret    CommonEnvCfgType = "Secret"
 	CommonEnvCfgTypePvc       CommonEnvCfgType = "PVC"
 )
+
+type AuthType string
```

Comment:
`aslan`, `reaper` and `wd` use this definition, so put them in a common pkg for reuse?